decidim-core/app/helpers/decidim/translations_helper.rb

```diff
@@ -11,8 +11,12 @@ module TranslationsHelper
     #
     # Returns a String with the translation.
     def translated_attribute(attribute)
+      return "" if attribute.nil?
+      return attribute unless attribute.is_a?(Hash)
+
       attribute.try(:[], I18n.locale.to_s).presence ||
         attribute.try(:[], current_organization.default_locale).presence ||
+        attribute.try(:[], attribute.keys.first).presence ||
         ""
```

Comment:
I think now you can change all those `attribute.try(:[], <method>).presence` with `attribute.<method>.presence`?

Reply:
👍